tests/Dfe.PlanTech.Infrastructure.Contentful.UnitTests/Content/Renderers/Models/PartRenderers/HyperlinkRendererTests.cs

```diff
@@ -82,7 +82,7 @@ public void Should_CreateLink_When_PassedValidData()
     public void Should_NotAddLink_When_MissingURI()
     {
         var renderer = new HyperlinkRenderer(new HyperlinkRendererOptions());
-        var rendererCollection = new RichTextRenderer(new NullLogger<IRichTextRenderer>(), new[] { renderer });
+        var rendererCollection = new RichTextRenderer(new NullLogger<RichTextRenderer>(), new[] { renderer });
```

Comment:
Probably not complexly more intense for the coimpiler but I'd ignore the .Net core 8.0 on this and just instantiate one interface and use it multiple times - upto you

Reply:
_In this instance_ it's just the typed `Logger` instance. I.e. this is now `Logger<RichTextRenderer>` instead of `Logger<IRichTextRenderer>`. 

I actually think it's better since it's more descriptive now from a logging perspective. If we had multiple classes of the interface `IRichTextRenderer`, it would be easier to tell exactly which one is logging the error, instead of them all having the same logger.